src/sentry/web/urls.py

```diff
@@ -131,6 +132,8 @@ def init_all_applications():
             debug_auth_views.DebugAuthConfirmIdentity.as_view()),
         url(r'^debug/auth-confirm-link/$',
             debug_auth_views.DebugAuthConfirmLink.as_view()),
+        url(r'^debug/icons/$',
+            DebugIconsView.as_view()),
```

Comment:
Sorry, and you'd have to import: `from django.views.generic import TemplateView` to get it.